specs/waku/v2/waku-filter.md

```diff
@@ -1,28 +1,34 @@
 ---
 title: Waku
-version: 2.0.0-alpha5
-status: Raw
+version: 2.0.0-beta1
+status: Draft
 authors: Oskar Thorén <oskar@status.im>
 ---
 
 # Table of Contents
 
-TODO
+- [Abstract](#abstract)
+- [Content filtering](#content-filtering)
+  * [Rationale](#rationale)
+  * [Protobuf](#protobuf)
+- [Changelog](#changelog)
+- [Copyright](#copyright)
+- [References](#references)
 
 # Abstract
```

Comment:
The spec reads a bit disjointed right now with abstract, content filtering, rationale.  Perhaps this can be rewritten to be more clear on what it does, what problem is being solved and how?

Reply:
It should explain that it is a lighter alternative to relay probably, and for BW sensitive devices